### ISSUE TITLE
Problem to get the google credentials

### ISSUE THREAD

**Issue by mdirren** [daball/Microsoft-Project-to-Google-Calendar#1]
Hello,

I have a problem on the "Transfer Tasks to Google Calendar" tab when I click on "Login..." after entered my gmail adress. Nothing happening.
Could you help me?

**Comment by daball** [comment]
First, let's make sure you are running the latest version of the application, which exhibits a different behavior than the prior version due to some Google API changes. The prior version no longer works at all due to Google API changes.

What normally occurs after you enter your Google Account email address, then click Login... for the first time, is that a web browser window pops up and asks you to Authorize the application to access your Calendars.

Once this authorization is complete, you can close the browser window and then proceed to select a Calendar from your list in Step 4 and complete the task by clicking Go... in Step 5. In future logins, the application simply pulls the authorization from your Windows user's AppData folder.

Let me know if this resolves your problem.

**Comment by mdirren** [comment]
Hi,

Thank you for your quick answer. Apparently it was a windows update to do. Probleme solved. However, the events are on a full day and I have some tasks of only 2 or 3 hours, is it possible to calibrate the tasks hour by hour rather than full day?

**Comment by daball** [comment]
You're welcome. Quick question: does Microsoft Project allow you to do this? It's been so long since I've used Project and I don't even have a license for it anymore, but I don't recall being able to set the actual date/time values as anything absolute.

Either way, it could very well become a feature. I'll revisit the project soon and see if I can add that feature. Sounds very useful to me. For my particular use case, it was unnecessary since we were in different activities for several days in length.

As far as implementation goes, would it be important to simply set a start date and time and use the length of each activity as occurring at some interval added to the start date/time, or would you prefer to be able to edit each activity's start date and time, or both?

**Comment by mdirren** [comment]
Indeed, in Microsoft Project 2010, the task dates are on a day (start day and finish day), you cannot specify an hour in the task information. But the duration can be with hour and on the graphic timeline you can place it to a particular hour. For exemple, I have a Task with a start day 29th May, finish day 29th May, duration 3h and on my timeline I placed it to begin at 9am. So I have a task from 8am to 11am the 29th May. Ideally I wish to have it from 9 to 11 the 29th May on my google calendar.

**Comment by cpwoods** [comment]
Hi i'm having the same issue, everything works perfectly until i try to login to google. The version i'm using i downloaded from your site - daball-Microsoft-Project-to-Google-Calendar-c5ace71.

thanks for any help
